llama_stack/apis/inference/inference.py

```diff
@@ -308,14 +308,50 @@ class CompletionResponseStreamChunk(BaseModel):
     logprobs: Optional[List[TokenLogProbs]] = None
 
 
+@json_schema_type
```

Comment:
small thing -- can you delete this and re-run the schema generator? Enums should not have this otherwise it creates an additional layer of indirection in the schema.